app/api/server/lib/users.js

```diff
@@ -28,3 +28,19 @@ export async function findUsersToAutocomplete({ uid, selector }) {
 		items: users,
 	};
 }
+
+/**
+ * Returns a new query object with the inclusive fields only
+ * @param {Object} query search query for matching rows
+ */
+export function getInclusiveFields(query) {
```

Comment:
I'm curious, why is this function needed?

Reply:
Because, otherwise we would get the error - **Both inclusive and exclusive fields are not allowed on the $project**

![img](https://user-images.githubusercontent.com/55396651/108212123-84d97480-7153-11eb-9380-2d532d9379b7.png)

Reply:
That makes sense, thanks!